### PR TITLE
Improve cheat buttons

### PR DESCRIPTION
**Pull request by smeas**
- Moved noclip toggle code into `CheatButtons`
- Added godmode [F3]
- Added HUD that shows active cheats

image

### CODE REVIEW

File: Assets/Scripts/Player/PlayerController.cs [hunk 1]
```diff
@@ -119,6 +119,20 @@ public class PlayerController : MonoBehaviour {
 	public float AirJumpLength => airJumpEndTime;
 	public float SpeedMultiplier { set => speedMultiplier = value; }
 
+#if UNITY_EDITOR
```
Comment (MrJackey) [inline Assets/Scripts/Player/PlayerController.cs]: Did you forget to add development build here?

**Review by MrJackey — CHANGES_REQUESTED** [review]
Just a tiny thing, otherwise :shipit:

File: Assets/Scripts/Player/PlayerController.cs (hunk 1 above)
Comment (smeas) [reply Assets/Scripts/Player/PlayerController.cs]: Yup, good catch!

**Review by Addehman — APPROVED** [review]
Handy!

**Review by MrJackey — APPROVED** [review]

**Pull request merged**